roles/node/defaults/main.yml

```diff
@@ -5,42 +5,79 @@
 avalanchego_version: 1.7.3
 avalanchego_binary_name: "avalanchego-linux-amd64-v{{ avalanchego_version }}.tar.gz"
 avalanchego_binary_url: "https://github.com/ava-labs/avalanchego/releases/download/v{{ avalanchego_version }}/{{ avalanchego_binary_name }}"
-avalanchego_bootstrap_db: ""
 
 # Install directories
 avalanchego_install_dir: /opt/avalanche/avalanchego
+avalanchego_releases_dir: "{{ avalanchego_install_dir }}/releases"
+avalanchego_current_dir: "{{ avalanchego_install_dir }}/current"
+avalanchego_plugins_dir: "{{ avalanchego_current_dir }}/plugins"
+avalanchego_vms_dir: /opt/avalanche/vms
+
+# DB directory
 avalanchego_db_dir: /var/lib/avalanchego/db
+
+# Configuration directories
 avalanchego_conf_dir: /etc/avalanche/avalanchego/conf
+avalanchego_chains_conf_dir: "{{ avalanchego_conf_dir }}/chains"
+avalanchego_vms_conf_dir: "{{ avalanchego_conf_dir }}/vms"
 avalanchego_certs_dir: /etc/avalanche/avalanchego/staking
+
+# Log directory
 avalanchego_log_dir: /var/log/avalanche/avalanchego
-avalanchego_vms_dir: /opt/avalanche/vms
 
-# User & network
+# User
 avalanchego_user: avalanche
+
+# Network
 avalanchego_use_static_ip: yes
+# Change this to public IP to accept API calls from the outside
 avalanchego_http_host: 127.0.0.1
 avalanchego_http_port: 9650
 avalanchego_staking_port: 9651
 
-# Logging
-avalanchego_log_level: info
-
-# Consensus & bootstrapping
+# Bootstrapping
 avalanchego_network_id: local
 avalanchego_use_existing_certs: yes
 avalanchego_local_certs_dir: "{{ playbook_dir }}/files/certs"
-avalanchego_snow_sample_size: 2
-avalanchego_snow_quorum_size: 2
 avalanchego_bootstrap_node_id: NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
+avalanchego_bootstrap_db: ""
 
-# Pre-funded account (only on local network)
-avalanche_prefunded_username: ewoq
-avalanche_prefunded_password: I_l1ve_@_Endor
-avalanche_prefunded_private_key: PrivateKey-ewoqjP7PxY4yr3iLTpLisriqt94hdyDFNgchSxGGztUrTXtNN
-
-# Subnets whitelist
+# Subnets
 avalanche_whitelisted_subnets: ""
 
 # VMs
 avalanchego_vms_install:
   - timestampvm=1.2.0
+
+# node.json
+avalanchego_node_json:
+  # Network
+  http-host: "{{ avalanchego_http_host }}"
+  http-port: "{{ avalanchego_http_port }}"
+  staking-port: "{{ avalanchego_staking_port }}"
+  api-admin-enabled: true
+  # Consensus
+  network-id: "{{ avalanchego_network_id }}"
+  snow-sample-size: 2
+  snow-quorum-size: 2
+  snow-mixed-query-num-push-vdr: 2
```

Comment:
I know we need to lower these for our small 5 nodes clusters but shouldn't we leave these by default in the role ? So in this case remove them.

Reply:
Yes I was wondering that also. I will create an issue and we can fix this later.